sample/README.md

```diff
@@ -1,30 +1,30 @@
-OpenTok Network Test sample app
+Vonage Video API Network Test sample app
 ===============================
 
 This sample uses the opentok-network-test-js module to check connectivity to
-OpenTok servers and test session quality.
+Vonage Video API servers and test session quality.
 
 ## Configuring the app:
 
 Make a copy of the /sample/src/js/configSample.js file, saving it to sample/src/js/config.js.
 Edit the properties in that file:
 
-* `apiKey` -- The API key corresponding to the OpenTok project the app uses.
+* `applicationId` -- The application ID corresponding to the Vonage Video API project the app uses.
```

Comment:
Please change `Vonage Video API project` to `Vonage Video API application` throughout.